dockers/docker-dhcp-server/dhcpservd.py

```diff
@@ -0,0 +1,521 @@
+            if "state" not in dhcp_server_entry or dhcp_server_entry["state"] != "enabled":
+                syslog.syslog(syslog.LOG_INFO, f"DHCP Server state for {dhcp_key} is not enabled, skip")
+                continue
+            if "mode" not in dhcp_server_entry:
+                syslog.syslog(syslog.LOG_WARNING, f"Missing dhcp mode setting for {dhcp_key}")
+            if dhcp_server_entry["mode"] == "PORT":
+                if dhcp_key not in vlan_interfaces:
+                    syslog.syslog(syslog.LOG_INFO, f"Cannot find interface IP for {dhcp_key}, skip")
+                for dhcp_interface_ip, ports in self.port_ips[dhcp_key].items():
+                    # Specify server id via option 54 of DHCP reply packet
+                    server_id = dhcp_interface_ip.split("/")[0]
+                    # Sepcify address lease time via option 51 of DHCP reply packet
+                    lease_time = int(dhcp_server_entry["lease_time"]) \
+                        if "lease_time" in dhcp_server_entry else DEFAULT_LEASE_TIME
+                    # Sepcify router ip via option 3 of DHCP reply packet
+                    gateway = dhcp_server_entry["gateway"] if "gateway" in dhcp_server_entry else server_id
+                    config_subnet = {
+                        "subnet": str(ipaddress.ip_network(dhcp_interface_ip, False)),
+                        "pools": [],
+                        "option-data": [
+                            {
+                                "name": "routers",
+                                "data": "{}".format(gateway)
+                            },
+                            {
+                                "name": "dhcp-server-identifier",
+                                "data": "{}".format(server_id)
+                            }
+                        ],
+                        "valid-lifetime": lease_time,
+                        "reservations": []
+                    }
+                    # Construct ip pools based on ip range.
+                    for port_name, ip_ranges in ports.items():
+                        if port_name in classes_port:
+                            class_set.append(client_class[port_name])
+                        for ip_range in ip_ranges:
+                            pool = {
+                                "pool": "{} - {}".format(str(ip_range[0]), str(ip_range[1])),
+                                "client-class": "{}:{}".format(hostname, port_name)
+                            }
+                            config_subnet["pools"].append(pool)
+                    # TODO Add customized options
+                    self.kea_config["Dhcp4"]["subnet4"].append(config_subnet)
+        if len(class_set) != 0:
+            self.kea_config["Dhcp4"]["client-classes"] = class_set
+
+        with open("/etc/kea/kea-dhcp4.conf", "w") as file:
+            json.dump(self.kea_config, file, indent=4, ensure_ascii=False)
+
+    def clear_table(self, db, table_name):
```

Comment:
This is general Redis operation. 
How about encapsulate a util class such as `DhcpDbConnector`? 
Try not mix util/cfg/daemon in a same place